process/management/commands/checker.py

```diff
@@ -17,6 +19,10 @@ class Command(BaseWorker):
     def __init__(self):
         super().__init__(self.worker_name)
 
+        if not settings.ENABLE_CHECKER:
+            self.logger.error("Checker is disabled in settings - see ENABLE_CHECKER value.")
+            sys.exit(1)
```

Comment:
Raise CommandError with the log message as the exception message: https://docs.djangoproject.com/en/3.2/howto/custom-management-commands/#command-exceptions